$OpenBSD: patch-configure,v 1.10 2012/08/16 17:03:18 eric Exp $

Use `Requires', not `Requires.private', because otherwise libraries
required for linking aren't exposed.

--- configure.orig	Sun Apr 29 14:49:59 2012
+++ configure	Mon Aug  6 00:31:32 2012
@@ -17696,10 +17696,7 @@ if test "x$PKG_CONFIG" = x; then
 	as_fn_error $? "pkg-config >= $PKGCONFIG_REQUIRED required but not found (http://pkgconfig.freedesktop.org/)" "$LINENO" 5
 fi
 
-case `$PKG_CONFIG --version` in
-0.?|0.?.?|0.1[0-7]|0.1[0-7].?) PKGCONFIG_REQUIRES="Requires"; ;;
-*) PKGCONFIG_REQUIRES="Requires.private"; ;;
-esac
+PKGCONFIG_REQUIRES="Requires"
 
 
 
